www/lendingbot.html

```diff
@@ -62,12 +62,12 @@
     <nav class="navbar navbar-default navbar-fixed-top">
         <div class="container">
             <div class="row">
-                <div class="col-xs-6">
+                <div class="col-xs-8">
                     <h4>
                         Poloniex Lending Bot
                     </h4>
                 </div>
-                <div class="col-xs-6" style="text-align:right;margin: 10px auto;" >
+                <div class="col-xs-4" style="text-align:right;margin: 10px auto;" >
```

Comment:
Yes, but otherwise "Poloniex Lending Bot" is on 2 lines (at least with 320x... and 360x... resolutions). IMHO date and time on 2 lines looks better, than name.